rosdep/python.yaml

```
@@ -8580,6 +8580,17 @@ python3-sqlalchemy-utils:
   nixos: [python3Packages.sqlalchemy-utils]
   opensuse: [python3-SQLAlchemy-Utils]
   ubuntu: [python3-sqlalchemy-utils]
+python3-sqlmodel:
+  debian:
+    '*':
+      packages: [python3-sqlmodel]
```

It's a little confusing, but the syntax here is slightly different:

```suggestion
    '*': [python3-sqlmodel]
```